releases/index.html

```diff
@@ -23,13 +24,59 @@
   | <a href="#14.2">14.2</a>
   | <a href="#14.1">14.1</a>
 </p>
-
 <div class="grid-10">
   <div class="version">
+    <a name="16.2"></a>
     <a name="latest"></a>
-
+    <h4>16.2 (latest)</h4>
+    <div class="section">
+      <h5>What's new in GoCD 16.2</h5>
+    <div class="section">
+      <h5>Features and bug fixes</h5>
+      <ul>
+        <li>
```

Comment:
Ok, I was thinking of it as something interesting for people to know what's going on. But, what you said makes sense and it's more about what has happened since the last release, that everyone needs to care about.